db/schema.rb

```diff
@@ -257,6 +257,7 @@
     t.string "urn"
     t.boolean "can_sponsor_skilled_worker_visa"
     t.boolean "can_sponsor_student_visa"
+    t.string "synonyms", default: [], array: true
```

Comment:
Could we add an index for this? I know we're not using it for search at the moment but just in case. `add_index :provider, : synonyms, using: 'gin'

Reply:
Discussed in the realm of real life, it not likely we will be using that as a lookup hence the need for indexes are not needed for the moment

Reply:
Sure lets add the index, but we need to be aware that this is an array and in future need to tailor queries appropriately.

Reply:
I've added it now [here](https://github.com/DFE-Digital/publish-teacher-training/pull/3363/commits/509f76f6278a84095fbab2389ee13874de6c77d9) but just thinking, can't we add it if we decide we need it later on to avoid confusion and over engineering?

Reply:
It's just one line to avoid having to remember later and potentially some headaches